flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/batch/sql/AggregateITCase.scala

```diff
@@ -367,6 +367,36 @@ class AggregateITCase(
     TestBaseUtils.compareResultAsText(result.asJava, expected)
   }
 
+  @Test
+  def testTumbleWindowAggregateWithCollectUnnest(): Unit = {
```

Comment:
Can we just not use COLLECT to construct the test samples? In general, a unittest should test one thing a time, not multiple. And also for the streaming case.